app/controllers/facility_statements_controller.rb

```diff
@@ -38,9 +38,9 @@ def create
       flash[:error] = text("no_selection")
     elsif @statement_creator.create
       @statement_creator.send_statement_emails
-      flash[:notice] = text(success_message, accounts: @statement_creator.formatted_account_list).html_safe
+      flash[:notice] = text(success_message, accounts: @statement_creator.formatted_account_list.html_safe).html_safe
```

Comment:
The description from the text_helper docs is helpful for me understanding this.

I just played around with it. It's looking to me like we don't need the outer html_self. I tried it both with `text` and `html` helpers.

I could be convinced otherwise, but might it be a little cleaner to move the inner `html_safe` into `StatementCreator`?